lib/shared/vercel-edge-config/src/edge-config.ts

```diff
@@ -7,12 +7,14 @@ export class EdgeConfigSource extends ConfigSource {
         super()
     }
 
-    async getConfig(
+    async getConfig<T extends boolean>(
         sdkKey: string,
         kind: 'server' | 'bootstrap',
         obfuscated: boolean,
+        lastModifiedThreshold?: string,
```

Comment:
where's this used?

Reply:
it's not used here but its part of the type signature for the abstract class